.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "kornia/geometry/nerf/kornia_nerf_jupyter"]
```

Comment:
let's not add this here, let's point to separated repos

Reply:
OK